armi/tests/__init__.py

```diff
@@ -193,12 +194,14 @@ def requires_fixture(fixtureFunction):
     Decorator to require a fixture to have been completed.
 
     Parameters
+    ----------
     ==========
     fixtureFunction : function without any parameters
         Fixture function is a function that has been decorated with ``fixture`` and is called prior to running
         the decorated function.
 
     Notes
+    -----
     =====
```

Comment:
above edit

Reply:
Damn.   Okay, thanks for that. Let me to do a big swipe and try to grab all of these.